spec/content_spec.ts

```diff
@@ -30,7 +30,7 @@ describe('content', () => {
     // wait a bit after sending keys to body
     function pressKey(key) {
         browser.actions().sendKeys(key).perform();
-        browser.sleep(50);
+        browser.sleep(1000);
```

Comment:
is it necessary to be that long? don't we have tests where `pressKey` is called multiple times?

Reply:
Probably not. I think I added this when testing a flaky test and if I remember correctly it didn't help. I'll try restoring it.